Dockerfile

```diff
@@ -0,0 +1,35 @@
+#!/bin/echo docker build . -f
+# -*- coding: utf-8 -*-
+
+FROM debian:12
```

Comment:
Maybe harmless or irrelevant but you configure:
runs-on: ubuntu-22.04 in build.yml

Reply:
ubuntu is the OS for github and debian is the reference OS that will be built from ubuntu, we're doing docker in docker ... but who cares :)